browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3548,6 +3548,12 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_VBAT_UNSUPPORTED_REGION_TEXT" desc="Vbat modal text for unsupported region">
       On <ph name="DUE_DATE">%1$s</ph>, we will be discontinuing support for existing virtual BAT balances. Unfortunately, there are no available custodians in your region (<ph name="REGION">%2$s</ph>) to withdraw your earnings. Until then, you can still contribute to your favorite creators.
     </message>
+    <message name="IDS_SPEEDREADER_MESSAGE_TITLE" desc="Message shown on the speedreader prompt.">
+      View page with speedreader?
+    </message>
+    <message name="IDS_SPEEDREADER_MESSAGE_BUTTON" desc="The label for the button to enable speedreader on current page.">
+      Enable
```

Comment:
maybe it's better to have `View` as it is done in upstream for a simplified page view? cc @rmcfadden3 for the words above. @wchen342 could you please post a picture here with the dialog, so Ryan has a context where does it appear?

Reply:
Thanks, @SergeyZhukovsky — yeah, a screenshot would help so I can understand full context. But, at a glance I'd say "View page with Speedreader?" makes intuitive sense.

Also note we've capitalized the word "Speedreader" elsewhere, so I've adjusted that above.

Reply:
@rmcfadden3 Here is a screenshot of the popup:

![Screenshot](https://user-images.githubusercontent.com/6406743/221823174-603da996-b687-456d-a8a5-87c74ad31fd1.png)

Reply:
Thanks, @wchen342 . Yeah, I'd say `View page with Spreedreader` works well here. And note the capitalization on "Speedreader"

Reply:
Yes I have made the "S" capitalized. Somehow it wasn't reflected in the conversation page I think.